parlai/agents/seq2seq/seq2seq.py

```diff
@@ -154,15 +154,6 @@ def build_model(self, states=None):
 
         if self.use_cuda:
             self.model.cuda()
-            if self.multigpu:
```

Comment:
this is so much cleaner now 😄

Reply:
Actually I should double check this one agaib

Reply:
yeah it's all good :-O